front/lib/api/assistant/configuration.ts

```diff
@@ -66,7 +66,7 @@ export async function getAgentConfiguration(
     limit: 1,
   });
 
-  if (!agent) {
+  if (!agent || agent.status === "archived") {
```

Comment:
Initially I added this constraint in the where of the query, but that means technically we can retrieve a former version that would be still active. It should never be the case, but this way it's more robust.